pkg/devfile/adapters/docker/component/adapter.go

```diff
@@ -195,7 +195,8 @@ func (a Adapter) Delete(labels map[string]string) error {
 	componentContainer := a.Client.GetContainersByComponent(componentName, containers)
 
 	if len(componentContainer) == 0 {
-		return errors.Errorf("the component %s doesn't exist", a.ComponentName)
+		log.Italicf("Component %s does not exist", componentName)
```

Comment:
Any reason for turning the error message into Italic? I guess we normally start error messages with a small letter.

Reply:
We do not want to error out on `odo delete` if the component is not there. We just want to print the information to the user. Any logger besides italic you think is better suited?

Reply:
I think Infof() is better suited for this task

Reply:
updated